nimbus-core/src/main/java/com/antheminc/oss/nimbus/domain/model/state/support/DefaultJsonParamSerializer.java

```diff
@@ -105,7 +107,11 @@ public void serialize(Param<?> p, JsonGenerator gen, SerializerProvider provider
 			
 			writer.writeObjectIfNotNull(K_MESSAGE, p::getMessages);
 			writer.writeObjectIfNotNull(K_VALUES, p::getValues);
+			writer.writeObjectIfNotNull(K_LABELS, p::getLabels);
 			
+			if (p.isCollection()) {
```

Comment:
@swethavemuri-anthem can this be moved within the existing if block for isCollection?